v2/src/modules/tokens/components/TokenHolders.vue

```diff
@@ -114,7 +114,7 @@ const erc721TokenHolders = computed<Erc721TokenOwnersType | undefined>(() => {
 })
 
 onErc721TokenHolderLoaded(({ data }) => {
-    if (data.getERC721TokenOwners) {
+    if (data?.getERC721TokenOwners) {
         if (!data.getERC721TokenOwners.owners || data.getERC721TokenOwners.owners.length < 1) {
             emit('isNft', false)
```

Comment:
you also have to add `emit('isNft', true)` somewhere. Otherwise right now when you go from ERC20 token page to nft the type stays the same